qutip/core/cy/qobjevo.pyx

```diff
@@ -292,9 +292,11 @@ cdef class QobjEvo:
 
         return out
 
-    def __call__(self, double t, dict args=None):
-        if args:
-            return QobjEvo(self, args=args)(t)
+    def __call__(self, double t, dict _args=None, **kwargs):
+        if _args is not None or kwargs:
+            if _args:
+                kwargs.update(_args)
+            return QobjEvo(self, args=kwargs)(t)
```

Comment:
In coefficient we also kept both. 
This kind ob backward compatibility is free and will help users pass from v4 to v5.

We already add `args` starting with "_" in a few places. Somewhere is the documentation we say that `args` cannot start with this, so there is minimal risk of name collision on `_args`. Sadly cython does not support positional only parameter...

Reply:
I personally still think that we should favour the `kwargs` syntax and  remove the `_args` argument when possible. I would not consider this backward compatibility completely free as it will potentially confuse new users (even with the documentation) and even QuTiP 4 users as `qobj(t, args={})` will not work

In any case, if you already discussed this with @hodgestar and you both agree on this, I will not delay this PR on being merged.

Reply:
The goal here is to try have at least one way to write code that works in both QuTiP 4 and 5 so that users might potentially support both for awhile without too much hardship.

Perhaps the coefficient function and QobjEvo __call__ situations are a bit different however. The coefficient functions are usually defined directly by the user, but I don't know how often users call QobjEvo objects directly if at all.

So maybe @AGaliciaMartinez is correct and we could simplify the signature of __call__ here?

Lots of little trade-offs to be made around these things to try make the lives of our users as painless as possible.